src/modules/kvs/commit.c

```diff
@@ -1164,59 +1164,34 @@ int commit_mgr_ready_commit_count (commit_mgr_t *cm)
 
```

Comment:
Not sure what the "atomic" comment refers to.  This appears to just be about how this function cleans up on error?

Better commit summary message would be useful, as well as expanded commit main message.

Reply:
Yeah, perhaps "atomic" is the wrong word.  In the past, data structures were modified on the fly as merging occurred.  Any error would lead to ```exit()```.  As we refactored ```exit()``` away and returned errors, we couldn't return half modified data structures.  So a number of functions were modified to be "atomic", where the data structure was successfully modified completely or not at all.  Maybe there's a better word than "atomic" here.

Reply:
Ah, well I would say in the context of processing transactions/commits, the "atomic" term is a bit overloaded :-)  Maybe just "fully clean up on error"?